include/spv_utils.h

```diff
@@ -71,6 +71,7 @@ class OpcodeIterator final {
 
   // Get the opcode from the first word of the instruction
   spv::Op GetOpcode() const;
+  uint32_t GetWordCount() const;
```

Comment:
@danginsburg Could we use `size_t` here?

Reply:
@snowzurfer Fixed in commit a2a86c4